btcjson/chainsvrresults.go

```diff
@@ -45,6 +45,13 @@ type GetBlockVerboseResult struct {
 	NextHash      string        `json:"nextblockhash,omitempty"`
 }
 
+// AddMultisigAddressResult models the data returned from the addmultisigaddress
+// command.
+type AddMultisigAddressResult struct {
+	Address      string `json:"address"`
+	RedeemScript string `json:"redeemScript"`
+}
+
```

Comment:
This format looks right based on an old [example from Gavin](https://gist.github.com/gavinandresen/3966071).

I think this means the whole wallet.go:`func (c *Client) AddMultisigAddress` code path was not working before, but it is not a big deal at this point since bchd is not aiming to support the wallet yet. Is that correct?